react-app/src/api/stakingAPI.ts

```diff
@@ -26,6 +27,9 @@ interface IStakingAPI {
     memo?: string
   ): Promise<SignedTx>;
   getUnstakingAmount(account: string): Promise<BigNumberCoin>;
+  getDelegatorStakes(
+    account: string
+  ): Promise<{ delegation: Delegation; balance: BigNumberCoin }[]>;
 }
```

Comment:
Seems there's a `DelegationResponse` type from `"cosmjs-types/cosmos/staking/v1beta1/staking"`
Maybe reimport the type in the models file and create our own
```
import { DelegationResponse as IDelegationResponse } from "cosmjs-types/cosmos/staking/v1beta1/staking";

interface DelegationResponse extends Omit<IDelegationResponse, "balance"> {
    balance: BigNumberCoin
}
```